agent/config.py

```
@@ -2,10 +2,12 @@
 import logging
 import secrets
 from dataclasses import dataclass, field
+from pathlib import Path
 from typing import List, Optional
```

Ruff (0.15.17)

[warning] `typing.List` is deprecated, use `list` instead

(UP035)